internal/pkg/process/analyze.go

```diff
@@ -80,7 +80,7 @@ func (a *Analyzer) Analyze(pid int, relevantFuncs map[string]interface{}) (*Targ
 	for _, dep := range a.BuildInfo.Deps {
 		depVersion, err := version.NewVersion(dep.Version)
 		if err != nil {
-			a.logger.Error(err, "error parsing module version")
+			a.logger.Error(err, "parsing dependency version", "dependency", dep)
```

Comment:
can you log what `dep.Version` was too? ie what failed to parse

Reply:
Yeah, this should log all that info. Let me see about getting an example.

Reply:
Example:

```terminal
{
  "level": "error",
  "ts": 1725648366.6897662,
  "logger": "Instrumentation.Analyzer",
  "caller": "process/analyze.go:82",
  "msg": "parsing dependency version",
  "dependency": {
    "Path": "github.com/go-logr/logr",
    "Version": "v1.4.2",
    "Sum": "h1:6pFjapn8bFcIbiKo3XT4j/BhANplGihG6tvd+8rYgrY=",
    "Replace": null
  },
  "stacktrace": "go.opentelemetry.io/auto/internal/pkg/process.(*Analyzer).Analyze\n\t/home/tyler/go/src/go.opentelemetry.io/auto/internal/pkg/process/analyze.go:82\ngo.opentelemetry.io/auto.NewInstrumentation\n\t/home/tyler/go/src/go.opentelemetry.io/auto/instrumentation.go:139\nmain.main\n\t/home/tyler/go/src/go.opentelemetry.io/auto/cli/main.go:108\nruntime.main\n\t/usr/lib/go/src/runtime/proc.go:272"
}
```

Reply:
Ah cool, yup didn't realize `dep` would be the whole object. My bad